### DISCUSSION TITLE
Mi pokédex

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#506] [Show and tell]
Buenas noches, así quedo mi pokédex

https://gloriacoralcerecedo.github.io/IntroJS/

**Comment by DanyVeneno** [comment]
Quedo muy bien Gloria ✨🍕

**Reply by GloriaCoralCerecedo** [reply]
Gracias!

**Reply by MikeDev206** [reply]
No inventes te quedo padrisimo!!! 
Soy tu fan!
